Bottom: 4fc266f11cf9b671c0c7021ad785ff8bc966fd92
Top:    d1ff4cd6f5bc0143ae738c09acd61376fd754061
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 22:01:44 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
